xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -31,18 +31,18 @@ class CGUIDialogVideoInfo :
 public:
   CGUIDialogVideoInfo(void);
   virtual ~CGUIDialogVideoInfo(void);
-  virtual bool OnMessage(CGUIMessage& message);
-  virtual bool OnAction(const CAction &action);
+  bool OnMessage(CGUIMessage& message) override;
+  bool OnAction(const CAction &action) override;
```

Comment:
Seems sane following those and with the core guidelines checkers it should be easy to stay consistent